ui/objects/semantic-ui/js/objects/editform/components/RestorationWork.jsx

```diff
@@ -80,17 +71,13 @@ export const RestorationWork = ({
                   toolbar:
                     "bold italic | bullist numlist | outdent indent | undo redo",
                   valid_elements: "strong,b,div,br,p,i,li",
```

Comment:
Why is it different than what was merged to oarepo-ui? It's not even used by onBlur hook

Reply:
This is another branch

Reply:
Nevertheless, it's the same issue as with: https://github.com/Narodni-repozitar/nr-docs/pull/117#discussion_r1634395472